src/components/policy/policy_external/src/policy_table/types.cc

```diff
@@ -2092,6 +2160,419 @@ void DeviceParams::SetPolicyTableType(PolicyTableType pt_type) {
+  if (!name.is_initialized()) {
+    return false;
+  }
+  if (!type.is_initialized()) {
+    return false;
+  }
+  if (!key.is_initialized()) {
+    return false;
+  }
+  if (!array.is_initialized()) {
+    return false;
+  }
+  if (!mandatory.is_initialized()) {
+    return false;
+  }
+  if (!params.is_initialized()) {
+    return false;
+  }
+  if (!since.is_initialized()) {
+    return false;
+  }
+  if (!until.is_initialized()) {
+    return false;
+  }
+  if (!removed.is_initialized()) {
+    return false;
+  }
+  if (!deprecated.is_initialized()) {
+    return false;
+  }
+  if (!minvalue.is_initialized()) {
+    return false;
+  }
+  if (!maxvalue.is_initialized()) {
+    return false;
+  }
+  if (!minsize.is_initialized()) {
+    return false;
+  }
+  if (!maxsize.is_initialized()) {
+    return false;
+  }
+  if (!minlength.is_initialized()) {
+    return false;
+  }
+  if (!maxlength.is_initialized()) {
+    return false;
+  }
+  return true;
+}
+
+void VehicleDataItem::ReportErrors(rpc::ValidationReport* report__) const {
+  if (!struct_not_empty()) {
+    rpc::CompositeType::ReportErrors(report__);
+  }
+  if (!name.is_valid()) {
+    name.ReportErrors(&report__->ReportSubobject("name"));
+  }
+
+  if (!ValidateNaming(std::string(name))) {
+    report__->set_validation_info(
+        "Invalid name valus [" + std::string(name) +
+        "]. It should not contain spaces or invalid chars.");
+  }
+  if (!type.is_valid()) {
+    type.ReportErrors(&report__->ReportSubobject("type"));
+  }
+  if (type.is_initialized() && !ValidateTypes()) {
+    report__->set_validation_info("Unknown type [" + std::string(name) +
+                                  "]: '" + std::string(type) + "'.");
+  }
+  if (!key.is_valid()) {
+    key.ReportErrors(&report__->ReportSubobject("key"));
+  }
+  if (!ValidateNaming(std::string(key))) {
+    report__->set_validation_info(
+        "Invalid key valus [" + std::string(key) +
+        "]. It should not contain spaces or invalid chars.");
+  }
+  if (!array.is_valid()) {
+    array.ReportErrors(&report__->ReportSubobject("array"));
+  }
+  if (!mandatory.is_valid()) {
+    mandatory.ReportErrors(&report__->ReportSubobject("mandatory"));
+  }
+  if (!params.is_valid()) {
+    params.ReportErrors(&report__->ReportSubobject("params"));
+  }
+  if (!since.is_valid()) {
+    since.ReportErrors(&report__->ReportSubobject("since"));
+  }
+  if (!until.is_valid()) {
+    until.ReportErrors(&report__->ReportSubobject("until"));
+  }
+  if (!removed.is_valid()) {
+    removed.ReportErrors(&report__->ReportSubobject("removed"));
+  }
+  if (!deprecated.is_valid()) {
+    deprecated.ReportErrors(&report__->ReportSubobject("deprecated"));
+  }
+  if (!minvalue.is_valid()) {
+    minvalue.ReportErrors(&report__->ReportSubobject("minvalue"));
+  }
+  if (!maxvalue.is_valid()) {
+    maxvalue.ReportErrors(&report__->ReportSubobject("maxvalue"));
+  }
+  if (!minsize.is_valid()) {
```

Comment:
Missing `maxsize`

Reply:
@jacobkeeler fixed in 76277de2068688672613bec5ac92a1beae5574dd